### DISCUSSION TITLE
How can I set the default theme to light?

### DISCUSSION THREAD

**Discussion by ashhhi** [binokochumolvarghese/lightbi-hugo#58] [Q&A]
I can not find the default setting code

**Comment by binokochumolvarghese** [comment]
Can you please elaborate the question and show the code?

**Comment by ashhhi** [comment]
Everytime I open the website, it will be dark mode, but I want to set it light default, I don't know in code where I can change it, my code is in https://github.com/ashhhi/ashhhi.github.io

**Comment by binokochumolvarghese** [comment]
The theme will take the default system settings. Please check the system mode settings.

**Comment by ashhhi** [comment]
Thanks! but my default system setting is light, still not working... did you write any code to default this?
